Add info on configuring an MI runtime (#1114)
wso2/docs-choreo-dev: en/docs/develop-components/manage-the-micro-integrator-runtime-version-in-your-integration-project.md

You can establish multiple runtime environments by utilizing updated MI instances. With a valid WSO2 subscription, you can also access and download updates to set up an updated MI server locally. This update process allows you to select a preferred update level as the runtime version. Consequently, you can run your integration on a runtime with the latest updates, ensuring optimal performance and access to new features.

### Configure a Micro Integrator runtime environment in Integration Studio

> If you are using [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/), follow these steps to set up a remote server instance and configure a Micro Integrator runtime environment to execute your integration project:

Reviewer: ```suggestion
If you are using [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/), follow these steps to set up a remote server instance with a Micro Integrator runtime environment to execute your integration project:
```

!!! info "Note"
    If you have not downloaded and installed WSO2 Integration Studio, you can follow the [installation instructions](installing-WSO2-Integration-Studio/) in the WSO2 Integration Studio documentation.

1. Launch WSO2 Integration Studio and click **Add Server** in the **Getting Started** view.
   ![Add server](../assets/img/develop-components/micro-integrator/add-server.png)
2. In the **Define a New Server** dialog, click to expand the `WSO2` directory and select **WSO2 Remote Server**.
   ![Define a remote server instance](../assets/img/develop-components/micro-integrator/define-a-remote-server-instance.png)

> 3. Enter the WSO2 Micro Integrator management URL in the **Server URL** field and click **Test Connection**.

Reviewer: Let's define what this is and how to find this?

Author: +1
Done.

   ![Set server URL](../assets/img/develop-components/micro-integrator/set-server-url.png)
4. Click **Finish**.
5. In the **Add and Remove** dialog, select the integration components to move from the left side to the right side.
6. Click **Add >**.
7. Click **Finish**. 

> Your integration project is now running in a remote server instance, configured with the Micro Integrator runtime environment you require.

Reviewer: ```suggestion
Your integration project is now running in a remote server instance with the Micro Integrator runtime environment you require.
```
